interfaces/mount/entry.go

```diff
@@ -168,3 +169,24 @@ func LoadFSTab(reader io.Reader) ([]Entry, error) {
 	}
 	return entries, nil
 }
+
+// SaveFSTab writes a list of entries to a fstab-like file.
+//
+// The supported format is described by fstab(5).
+//
+// Note that there is no support for comments, both the LoadFSTab function and
+// SaveFSTab just ignore them.
+//
+// Note that there is no attempt to use atomic file write/rename tricks. The
```

Comment:
@chipaca Well, exactly. /var/lib/snapd/mount is full of .fstab files. I do hope we use those same helpers to write content there instead of having two different ways to write those files. So it's not ephemeral, and should definitely be written atomically.

Despite that, I am calm. We just need to make sure we're not fine with corrupt state lying around.

cc @zyga